library/formatting.inc.php

```diff
@@ -216,3 +216,24 @@ function oeFormatAge($dobYMD, $nowYMD = '', $format = 0)
 
     return $age;
 }
+
+
+/**
+ * Returns the complete formatted stae string according the global date format
+ * @param $timestamp
+ * @return bool|string
+ */
+function oeFormatLongDate($timestamp)
+{
+    if(!$timestamp){
+        $timestamp = date('Y-m-d H:i');
+    }
+    // assume input is yyyy-mm-dd
+    if ($GLOBALS['date_display_format'] == 1) {      // mm/dd/yyyy, note year is added below
+        $newDate = date('m-d-Y H:i',$timestamp);
```

Comment:
To help out, let me know a quick example of what the $timestamp parameter will look like.